fix: community page update

vinckr opened the pull request:
<!--
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

This text will be included in the changelog. If applicable, include links to documentation or pieces of code.
If your change includes breaking changes please add a codeblock documenting the breaking change:

```
BREAKING CHANGES: This patch changes the behavior of configuration item `foo` to do bar. To keep the existing
behavior please do baz.
```
-->

## Related Issue or Design Document

<!--
If this pull request

1. is a fix for a known bug, link the issue where the bug was reported in the format of `#1234`;
2. is a fix for a previously unknown bug, explain the bug and how to reproduce it in this pull request;
2. implements a new feature, link the issue containing the design document in the format of `#1234`;
3. improves the documentation, no issue reference is required.

Pull requests introducing new features, which do not have a design document linked are more likely to be rejected and take on average 2-8 weeks longer to
get merged.

You can discuss changes with maintainers either in the Github Discusssions in this repository or
join the [Ory Chat](https://www.ory.sh/chat).
-->

## Checklist

<!--
Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
-->

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
      vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the
      maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

## Further comments

<!--
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution
you did and what alternatives you considered, etc...
-->

vinckr commented:
update to the community page, fyi @jfcurran @BlueAcidFrog 

I basically 
- removed a lot of fluff, 
- added links for the new example repo and archive,
- removed the listings for examples and articles

jfcurran commented on src/page-content/content-community.tsx:
```diff
@@ -21,14 +21,14 @@ export const seo: SeoProps = {
   buttons: [
     {
-      ctaTitle: 'Join the Chat on Slack',
+      ctaTitle: 'On this page we laid out some easy steps to get you started. 👇',
```
```suggestion
      ctaTitle: 'Join the community',
```

jfcurran replied:
Sounds good, but keep the CTA short - 30 characters max

jfcurran commented on src/page-content/content-community.tsx:
```diff
@@ -21,14 +21,14 @@ export const seo: SeoProps = {
+  title: 'Welcome to the Ory Community',
   description:
-    'You are the backbone of Ory. Members are coming from all around the world, from huge enterprises or small side projects. We may have many differences, but we all share the same goal: To build an open and free authentication and authorization standard for everyone.',
+    'The Ory community offers tutorials, examples, discussions, and answers to questions on a wide range of topics. Where to start?',
```
```suggestion
    'The Ory community offers tutorials, examples, discussions, and answers to questions on a wide range of topics.',
```

jfcurran replied:
agree, it makes sense when either the cta does a good job explaining or when the description is improved. either works 👍

jfcurran reviewed (commented):
How about simplifying the CTA a bit? 👇

vinckr commented:
- Fixed the CTA according to your suggestions, thanks @jfcurran 
- Moved the duplicate entry "community" under "solutions" to "developers" and named it "github" (at it leads to our github org page) // this also is the same structure as in the footer now.

BlueAcidFrog reviewed (approved):
Good to go
